src/lib/coda_base/pending_coinbase.ml

```diff
@@ -366,16 +366,9 @@ module T = struct
   (* the arguments to Sparse_ledger.Make are all versioned; a particular choice of those
      versions yields a version of the result
    *)
-  module Merkle_tree = struct
-    module Stable = struct
-      module V1 =
-        Sparse_ledger_lib.Sparse_ledger.Make
-          (Hash.Stable.V1)
-          (Stack_id.Stable.V1)
-          (Stack.Stable.V1)
-      module Latest = V1
-    end
-  end
+  module Merkle_tree =
```

Comment:
I still think the `Merkle_tree` result should be versioned. If you pass the functor different versions for `Hash`, etc., you'll get a different result. With different arguments, `Merkle_tree.V1.t` could change.